new-dti-website-redesign/src/app/products/page.tsx

```diff
@@ -12,6 +15,24 @@ export default function Products() {
         <h1>PRODUCTS</h1>
         <p className="mt-2">This is the PRODUCTS page</p>
       </section>
+
+      <SectionSep />
+
+      {products.map((product, index) => (
+        <>
+          <Product
+            key={index}
+            image={product.image}
+            imageAlt={product.imageAlt}
+            name={product.name}
+            description={product.description}
+            link={product.link}
+            comingSoon={product.comingSoon}
+          />
```

Comment:
You can destructure `products`.
```suggestion
          <Product key={index} {...product} />
```